src/screens/Booking/Booking.tsx

```diff
@@ -0,0 +1,53 @@
+// @ts-nocheck
```

Comment:
we shouldn't use it here (:

let fix the ts warning in the file src/screens/Booking/Booking.styles.ts